vsphere/resource_vsphere_compute_cluster.go

```diff
@@ -133,11 +133,12 @@ func resourceVSphereComputeCluster() *schema.Resource {
 				Description: "The managed object ID of the datacenter to put the cluster in.",
 			},
 			"host_system_ids": {
-				Type:        schema.TypeSet,
-				Optional:    true,
-				MaxItems:    64,
-				Description: "The managed object IDs of the hosts to put in the cluster.",
-				Elem:        &schema.Schema{Type: schema.TypeString},
+				Type:          schema.TypeSet,
+				Optional:      true,
+				MaxItems:      64,
+				Description:   "The managed object IDs of the hosts to put in the cluster.",
+				Elem:          &schema.Schema{Type: schema.TypeString},
+				ConflictsWith: []string{"host_managed"},
```

Comment:
I actually didn't know about this option, _nice_!

Reply:
It is certainly a nice one!